wasm.md

```diff
@@ -1096,16 +1058,19 @@ Currently there is only one possiblt value for it which is "funcref".
 The allocation of a new `tableinst`.
 Currently at most one table may be defined or imported in a single module.
 The only allowed `TableElemType` is "funcref", so we ignore this term in the reducted sort.
+The table values are addresses into the store of functions.
 
 ```k
     syntax Defn      ::= TableDefn
-    syntax TableDefn ::= "(" "table"     OptionalId Limits TableElemType ")"
-                       | "(" "table"     OptionalId        TableElemType "(" "elem" ElemSegment ")" ")"
+    syntax TableType ::= Limits TableElemType
```

Comment:
I prefer to put the definition of `TableType`, `TypeUse`, `TableType` and `TextGlobalType` together because they are not only used in their own definition, but are also used elsewhere in the program. Putting them together in another section makes people know that "if you change this, you will not only influence this section, but you will also influence somewhere else"

Reply:
I'm not sure I follow. What's the connection between `TypeUse` on the one hand and `TextGlobalType` and `TableType` on the other. Do you mean a section about all the "types" `GlobalType`, `MemType`, `FuncType` and `TableType`?

Reply:
Yes I just mean having a separated section for all the `types`

Reply:
Good, I'll make a note of that for refactoring. Probably best to avoid doing big file rearrangements in this PR.